codegen/templates/endpoint.tmpl

```diff
@@ -50,20 +52,25 @@ func Handle{{title .Name}}Request(
 	{{end}}
 
 	headers := map[string]string{}
+	// TODO(sindelar): Add optional headers in addition to required.
+	for _, k := range {{.Headers | printf "%#v" }} {
```

Comment:
This is not what .Headers means. We already have `req.CheckHeaders()`

We want to get all headers here, all of them.